seminars.html

```diff
@@ -0,0 +1,130 @@
+                            </p>
+                        </li><br>
+                        <li class="leading-block"><span class="blog-post-date">03 December, 2020 - 12pm PDT/5pm GMT</span>
+                            <h4>Extending C++ for Heterogeneous Quantum-Classical Computing by Alex McCaskey</h4>
+                            <p>
+                                We present <strong><a href="https://qcor.ornl.gov/">qcor</a></strong> - a language extension to C++ and compiler implementation that enables heterogeneous quantum-classical programming, compilation, and execution in a single-source context. Our work provides a first-of-its-kind C++ compiler enabling high-level quantum kernel (function) expression in a quantum-language agnostic manner, as well as a hardware-agnostic, retargetable compiler workflow targeting a number of physical and virtual quantum computing backends. qcor leverages novel Clang plugin interfaces and builds upon the <a href="https://github.com/eclipse/xacc"> XACC</a> system-level quantum programming framework to provide a state-of-the-art integration mechanism for quantum-classical compilation that leverages the best from the community at-large. qcor translates quantum kernels ultimately to the XACC intermediate representation, and provides user-extensible hooks for quantum compilation routines like circuit optimization, analysis, and placement. This work details the overall architecture and compiler workflow for qcor, and provides a number of illuminating programming examples demonstrating its utility for near-term variational tasks, quantum algorithm expression, and feed-forward error correction schemes.
+                            </p>
+                            <p>
+                                <strong>Alex McCaskey</strong> is a research scientist in the Computer Science and Mathematics Division at Oak Ridge National Laboratory. He serves as the Software Lead for the Quantum Computing Institute at ORNL and is the Project Lead for the XACC quantum framework and the QCOR quantum-classical C++ compiler. He received his Masters in Physics from Virginia Tech and BS degrees in Physics and Mathematics from the University of Tennessee.  
+                            </p>
+                        </li><br>
+                    </ul>
```

Comment:
Currently this amount of text is a bit overwhelming and could get very long quickly as we add talks. There's a few ways to deal with this, but perhaps it is easiest just to have titles and speaker names that hyperlink to the project and the speaker?

Am open to other suggestions

Reply:
Basically I need a place to put their full abstract and bio... I'm gonna try some collapsible details elements and see how that works